src/init.cpp

```diff
@@ -228,7 +228,6 @@ void PrepareShutdown()
 #endif
     GenerateBitcoins(false, 0, Params(), *g_connman);
```

Comment:
If `fGenerate` (first argument of `GenerateBitcoins()`) is `false`, `connman` argument is not used.